libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -191,18 +192,33 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("2_BUS",  20, AP_Airspeed, param[1].bus, 1),
     
+    // @Param: _BLEND
+    // @DisplayName: Blending of airspeed sensors
+    // @Description: Sets the configuration for enable the blending of multiple airspeed sensors
+    // @Values: 0:Blend disabled,1:Full blending
+    // @User: Advanced
+    AP_GROUPINFO("_BLEND",  21, AP_Airspeed, blend_conf, 0),
+
     AP_GROUPEND
 };
 
-
 AP_Airspeed::AP_Airspeed()
 {
     for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
         state[i].EAS2TAS = 1;
+        state[i].blend_healthy = true;
     }
     AP_Param::setup_object_defaults(this, var_info);
+    
+    _ahrs = nullptr;
 }
 
+AP_Airspeed::AP_Airspeed(AP_AHRS &ahrs)
+{
+    AP_Airspeed();
+    
+    _ahrs = &ahrs;
```

Comment:
Please this new second constructor and assign _ahrs like it is in other libraries. ahrs
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_TECS/AP_TECS.h#L32

_ahrs should never be null. We also now have a singleton for it so this is all unnecessary but I think that singleton replacement will happen globally by another PR.

Reply:
I tried such method.
The problem lies within /Tools/Replay/ and /examples/ that simply does not have any AHRS

Reply:
AHRS is a singleton now and can be accessed that way.

Reply:
@WickedShell Can you provide a hint? That concept is new for me in AP.

Reply:
It went in in #7473 for a specific example here is how AP_ADSB changed: https://github.com/ArduPilot/ardupilot/pull/7473/commits/0dff8e87810e3ab476e6d419af88d993562a3c1a

Reply:
Thank you!